Formula/openjdk.rb

```diff
@@ -0,0 +1,65 @@
+class Openjdk < Formula
+  desc "Development kit for the Java programming language"
+  homepage "https://openjdk.java.net/"
+  url "https://hg.openjdk.java.net/jdk-updates/jdk13u/archive/jdk-13.0.1+9.tar.bz2"
+  version "13.0.1+9"
+  sha256 "97328e767bc5f47b097ec0e9d88a6a650e60c448dbaba2e835284a2bf5594eb5"
+
+  keg_only :provided_by_macos
+
+  depends_on "autoconf" => :build
+
+  resource "boot-jdk" do
+    url "https://download.java.net/java/GA/jdk12.0.2/e482c34c86bd4bf8b56c0b35558996b9/10/GPL/openjdk-12.0.2_osx-x64_bin.tar.gz"
+    sha256 "675a739ab89b28a8db89510f87cb2ec3206ec6662fb4b4996264c16c72cdd2a1"
+  end
```

Comment:
Shouldn't `boot-jdk` version be the same as the formula version?

Reply:
It's recommended building JDK n with JDK (n - 1).

Reply:
Thanks for the explanation!